docs/README.md

```diff
@@ -0,0 +1,15 @@
+Detail contracts decumetation can be found [here](ref/README.md)
```

Comment:
Detail -> Detailed, decumetation -> documentation.
Please use a spelling checker in your IDE.